data_managers/data_manager_build_kraken_database/tool_dependencies.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0"?>
+<tool_dependency>
+    <set_environment version="1.0">
+        <environment_variable action="set_to" name="KRAKEN_SCRIPT_PATH">$REPOSITORY_INSTALL_DIR</environment_variable>
```

Comment:
This is a good thing going forward, but I do seriously wonder if it is worth needlessly increasing the minimal Galaxy version that would then be required to use this tool. 

e.g. are any other newer/same aged features used?